### PR TITLE
Answer text changes saved graphql

### PR DESCRIPTION
**Pull request by bennpearson**
### What is the context of this PR?
Answer text input working with graphql

### How to review 
Check answer text component is working with graphql and that all tests pass

### CODE REVIEW

File: src/containers/QuestionnaireDesignPage/QuestionnaireDesignPage.js [hunk 1]
```diff
@@ -16,49 +16,60 @@ export class QuestionnaireDesignPage extends Component {
...
+  handleAddAnswer = type => {
+    this.props.onAddAnswer(type);
   };
 
   handleBlur = focused => {
     this.setFocused(focused);
 
-    switch (this.state.focused) {
-      case "section":
+    switch (true) {
```
Comment (WickyNilliams) [inline src/containers/QuestionnaireDesignPage/QuestionnaireDesignPage.js]: I think this switch statement would be clearer as a series of if/else if/else.

File: src/components/Answers/TextAnswer.js [hunk 2]
```diff
@@ -2,20 +2,21 @@ import React from "react";
 import PropTypes from "prop-types";
 import CustomPropTypes from "custom-prop-types";
 import { Field, Input } from "components/Forms";
-import SeamlessInput from "./SeamlessInput";
-import SeamlessTextArea from "./SeamlessTextArea";
+import SeamlessInput from "components/QuestionnaireDesign/SeamlessInput";
```
Comment (samiwel) [inline src/components/Answers/TextAnswer.js]: Any reason why these have been moved back? Discussed with @hamishtaplin previously that SeamlessInput should really be it's own component, not nested down inside QuestionnaireDesign.

File: src/components/Answers/TextAnswer.js [hunk 3]
```diff
@@ -2,20 +2,21 @@ import React from "react";
...
+import SeamlessTextArea from "components/QuestionnaireDesign/SeamlessTextArea";
```
Comment (samiwel) [inline src/components/Answers/TextAnswer.js]: Same as above.

**Comment by bennpearson** [comment]
[Add/edit text field answer (5)](https://trello.com/c/mQZxAPTs/71-add-edit-text-field-answer-5)

**Pull request merged**